src/engine/Server.cpp

```diff
@@ -67,7 +69,56 @@ Awaitable<void> Server::process(
   ad_utility::Timer requestTimer;
   requestTimer.start();
 
-  auto filenameAndParams = ad_utility::UrlParser::parseTarget(request.target());
+  // Parse the payload from a GET or POST request.
+  LOG(DEBUG) << "Request method: \"" << request.method() << "\""
+             << ", target: \"" << request.target() << "\""
+             << ", body: \"" << request.body() << "\"" << std::endl;
+  auto filenameAndParams = [&request]() {
+    if (request.method() == http::verb::get) {
+      // For a GET request, `request.target()` yields the part after the domain,
+      // which is a concatenation of the path and the query string (the query
```

Comment:
For me as a human the usage of the term `query` (or `query string`) for 
the GET parameters with `?` and `&` connections is a little bit confusing, because we use `query` typically as a synonym for SPARQL query`